projects/plugins/boost/app/modules/cache/Page_Cache.php

```diff
@@ -104,6 +114,55 @@ private function add_wp_cache_define() {
 		}
 	}
 
+	/*
+	 * Removes the advanced-cache.php file and the WP_CACHE define from wp-config.php
+	 * Fired when the plugin is deactivated.
+	 */
+	public function deactivate() {
+		$this->delete_advanced_cache(); // how do we handle errors?
+		$this->delete_wp_cache_constant(); // how do we handle errors?
```

Comment:
Removal probably works fine for most sites. Occasionally, I'd see complaints about WP Super Cache leaving behind files but that might be because people deleted the plugin directory. Not much we can do to stop them doing that.

Reply:
I just spotted where [WPSC handles this](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/super-cache/wp-cache.php#L3733), and it sends an email to the site owner if files have been left behind, giving instructions on what to do.

Reply:
I'm going to half-implement this. If I send the actual email, we'll end up making a monstrosity of a thing with proper Jetpack formatting and all that..